### ISSUE TITLE
CPU only support even when theres a GPU (suggestion)

### ISSUE THREAD

**Issue by boringhalfman** [Pavelevich/llm-checker#115]
hi! I wanted to suggest this since sometimes a users GPU isn't powerful enough for running a model compared to their CPU and RAM.

**Comment by signerless** [comment]
Implemented and merged in #119.

You can now force CPU/RAM execution even when a GPU is detected:

```bash
llm-checker recommend --cpu-only
llm-checker ai-run --cpu-only --models qwen2.5:7b
llm-checker gpu-plan --cpu-only --model-size 14GB
```

For a shell-wide default, set `LLM_CHECKER_CPU_ONLY=1`. Programmatic callers can use `{ cpuOnly: true }`, with explicit `{ cpuOnly: false }` taking precedence over the environment.

CPU-only mode keeps the detected GPU visible as diagnostic inventory, but sets active VRAM and GPU count to zero and bases fit, tier, scoring, planning, and speed estimates on the CPU plus system RAM. Real Ollama requests now force `num_gpu: 0`, including AI Run, AI Check, probes, and MCP execution/benchmark tools. Accelerator-only MLX, Metal bonuses, and GPU speculative-decoding paths are excluded while the mode is active.

Validation completed with 57/57 local tests plus green CI on Node 18, 20, and 22, the policy gate, and Windows smoke tests. Normal GPU behavior and the default safe-model filter were regression-tested as well.

**Comment by boringhalfman** [comment]
thanks!
